### ISSUE TITLE
station airport info are wrong

### ISSUE THREAD

**Issue by mohkhz2001** [avwx-rest/avwx-api#25]
hello
thank you for this API but have wrong info
example:
OIIE -> have two runway (11R/29L && 11L/29R) but this api show me the just on runway

**Comment by eagleDiego** [comment]
```
==========
Disclosure
==========
I have a direct interest in the company and service mentioned below!
```

It is very difficult to maintain such data. I believe most of this database comes from the last issue of DAFIF, but a lot has changed since then.
I work for [aviowiki](https://app.aviowiki.com), which includes also detailed [runway information](https://www.aviowiki.com/docs/airports/data-models/runway/). We have a team of data curators that look after updates, and I can tell you it is not an easy task.

If you are looking at something to use in a commercial product, let me know and I can send you more details.
There are also some [free endpoints](https://www.aviowiki.com/docs/airports/free-api-endpoints/) that can be used for no charge, but runways details are not included.

As I discussed a while ago with @flyinactor91 we have plans to contribute to this project with updated information from our database, but we won't be able to release the entire database of runways for sure, due to commercial aspects.

**Comment by flyinactor91** [comment]
Airport info is compiled from ourairports.com and other sources. I'll double check the info and update there.

Should probably add that to the README.

@eagleDiego I'll find your email again. Made some changes since then that should better support a collaboration.

**Comment by flyinactor91** [comment]
Looked up the issue. The compilation step doesn't include closed runways as this is 11R's current status. Wikipedia has this runway closed as well:

https://ourairports.com/airports/OIIE/runways.html
https://en.wikipedia.org/wiki/Tehran_Imam_Khomeini_International_Airport

Has the runway opened since?

**Comment by eagleDiego** [comment]
The runway is open subject to certain conditions. A bit similar to EGKK's 08L:

From Iran AIP OIIE AD 2.12.12:
>RWY  29L/11Ris  usable  for  take off and landing when:
>-RWY 29R/11L is not AVBL
>-No ACFT OPS on TWY A
>-No parked ACFT in the north side of Cargo apron
>-No  parked  ACFT CAT D (except A300B4/A300-600) in  the  north side of passenger apron.

**Comment by mohkhz2001** [comment]
@eagleDiego 
thank you for response to this issue. I checked the [https://app.aviowiki.com/](url)  and it's good and powerfull website so generally it's better than other.
when I search OIAM airport , in runway info show  tell me have 4 runways that 2 of them are Asphalt (it's okay) and 2 other runways are "Contribute!". what does it mean?

**Comment by eagleDiego** [comment]
@mohkhz2001 "Contribute!" means we don't have some properties, and that users can contribute to the data by [signing up](https://app.aviowiki.com/registration) for a free account.
Contributions are then checked by the Data Team before being merged in the public dataset.

In this specific case, our Data Team is still going through the airports in Iran after we imported the data from the AIP, to eliminate some runways that were imported from an "official" source, and were (unsurprisingly) wrong.
I already corrected this case. Thanks for pointing it out!
